src/Data/Conversion.php

```diff
@@ -74,6 +76,49 @@ public static function objectArray($object)
         return false;
     }
 
+    /**
+     * Convert the bit into bytes.
+     *
+     * @param int $size The value that you want provided
+     * @param int $pre  Round the value default 2
+     *
+     * @since 3.0.0
+     *
+     * @return mixed
+     */
+    public static function bitToBytes($size, $pre = 2)
+    {
+        $base = log($size) / log(1024);
+        $suffix = Arrays::arrayChangeCaseValue(['b', 'k', 'm', 'g', 't', 'p', 'e', 'z', 'y'], CASE_UPPER);
+        $f_base = (floor($base) > 8) ? 8 : floor($base);
+
+        return round(pow(1024, $base - floor($base)), $pre).$suffix[$f_base];
+    }
+
+    /**
+     * Convert the views to relative unit.
+     *
+     * @param int $n   Views.
+     * @param int $sep Seperator.
```

Comment:
This is weird.

The `$sep` is `integer` type annotation, but the default value is `,` defined in `views` method arguments.

You should consider that.